app/controllers/account_details.js

```diff
@@ -111,7 +113,7 @@ export default Ember.Controller.extend({
       user_attributes: {
         first_name: user.get("firstName"),
         last_name: user.get("lastName"),
-        mobile: user.get("mobile"),
+        mobile: this.mobileParam(),
```

Comment:
Can we pass user instead of getting user again in mobileParam?

Reply:
Sure 👍